src/components/overlays/Tooltip/Tooltip.stories.tsx

```diff
@@ -1,48 +1,70 @@
+import { Story, ComponentMeta } from '@storybook/react';
+import { within, userEvent, waitFor } from '@storybook/testing-library';
+import { expect } from '@storybook/jest';
 import { Tooltip } from './Tooltip';
 import { Button } from '../../actions';
-import { TooltipTrigger } from './TooltipTrigger';
-import { TooltipProvider } from './TooltipProvider';
-import { Block } from '../../Block';
+import { TooltipTrigger, CubeTooltipTriggerProps } from './TooltipTrigger';
+import { CubeTooltipProviderProps, TooltipProvider } from './TooltipProvider';
 import { baseProps } from '../../../stories/lists/baseProps';
 
 export default {
   title: 'UIKit/Overlays/Tooltip',
   component: Tooltip,
   parameters: {
+    layout: 'centered',
     controls: {
       exclude: baseProps,
     },
   },
-};
+} as ComponentMeta<typeof Button>;
 
-const Template = (args) => (
+const Template: Story<CubeTooltipTriggerProps> = (args) => (
   <TooltipTrigger {...args}>
-    <Button margin="8x 18x">Hover to show a tooltip</Button>
+    <Button>Hover to show a tooltip</Button>
     <Tooltip>Tooltip content</Tooltip>
   </TooltipTrigger>
 );
 
-const ViaProviderTemplate = (args) => (
+const ViaProviderTemplate: Story<CubeTooltipProviderProps> = (args) => (
   <TooltipProvider title="Tooltip content" {...args}>
-    <Button margin="8x 18x">Hover to show a tooltip</Button>
```

Comment:
It's for testing purposes. If you change the direction the tooltip will be cropped.

Reply:
no, it won't, check the storybook